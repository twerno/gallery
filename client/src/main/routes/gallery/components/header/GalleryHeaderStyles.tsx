import styled from 'styled-components';

const SearchForm = styled.form`
    width: 100%;
    display: flex;
    flex-direction: row;
    padding: 10px 10px;
    background-color: white;
`;


export default {
    SearchForm
}