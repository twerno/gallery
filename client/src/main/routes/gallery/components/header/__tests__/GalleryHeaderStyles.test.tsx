import '@testing-library/jest-dom/extend-expect';
import 'jest-styled-components';
import * as React from 'react';
import { componentBasicTests } from 'test/componentBasicTests';
import GalleryHeaderStyles from '../GalleryHeaderStyles';

describe('GalleryHeaderStyles', () => {

    componentBasicTests(
        "SearchForm",
        <GalleryHeaderStyles.SearchForm />
    );

});

