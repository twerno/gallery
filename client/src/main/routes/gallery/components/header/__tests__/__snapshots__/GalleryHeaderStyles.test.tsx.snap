// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GalleryHeaderStyles [SearchForm] Component basic tests matches the snapshot 1`] = `
.c0 {
  width: 100%;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  padding: 10px 10px;
  background-color: white;
}

<form
  className="c0"
/>
`;
